peru/runtime.py

```diff
@@ -76,7 +76,7 @@ def _set_paths(self, args, env):
                 'If the --file or --sync-dir is set, '
                 'the other must also be set.')
         else:
-            basename = explicit_basename or parser.DEFAULT_PERU_FILE_NAME
+            basename = explicit_basename or parser.DEFAULT_PERU_FILENAMES[0]
```

Comment:
This (and `find_project_file` below) is where you'd need to loop over the list of names, if we wanted to support more than one. Since this line is effectively unchanged here, I don't think this PR has actually added support for the other names. Were you able to test this?